docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -169,30 +170,32 @@ You will also see in the Dev UI console something similar to:
 2021-07-19 17:58:11,674 INFO  [io.qua.oid.dep.dev.key.KeycloakDevConsolePostHandler] (security-openid-connect-quickstart-dev.jar) (DEV Console action) Result: 200
 ----
 
-A token is acquired from Keycloak using a `password` grant and is sent to the service endpoint.
+A token is acquired from Keycloak by using a `password` grant and is sent to the service endpoint.
 
-==== Client Credentials Grant
+==== Client credentials grant
 
-If you set `quarkus.oidc.devui.grant.type=client` then a `client_credentials` grant will be used to acquire a token, with the page showing no `User` field in this case:
+If you set `quarkus.oidc.devui.grant.type=client`, then a `client_credentials` grant is used to acquire a token, with the page showing no *User* field in this case:
 
 image::dev-ui-keycloak-client-credentials-grant.png[alt=Dev UI OpenID Connect Keycloak Page - Client Credentials Grant,role="center"]
 
-Select a realm, enter the client id and secret, a relative service endpoint path, click on `Test Service` and you will see a status code such as `200`, `403`, `401` or `404` printed.
+Select a realm, enter the client ID and secret, a relative service endpoint path, and click *Test service*. It returns a status code, such as `200`, `403`, `401`, or `404`.
 
 [[develop-web-app-applications]]
-=== Developing OpenID Connect Web App Applications
+=== Developing OpenID Connect web-app applications
 
-If you develop a xref:security-oidc-code-flow-authentication.adoc[Quarkus OIDC web-app application], then you should set `quarkus.oidc.application-type=web-app` in `application.properties` before starting the application.
+To develop a xref:security-oidc-code-flow-authentication.adoc[Quarkus OIDC web application], set `quarkus.oidc.application-type=web-app` in `application.properties` before starting the application.
 
-You will see a screen like this one:
+Starting the application displays a screen similar to this one:
 
 image::dev-ui-keycloak-sign-in-to-service.png[alt=Dev UI OpenID Connect Keycloak Sign In,role="center"]
 
-Set a relative service endpoint path, click on `Sign In To Service` and you will be redirected to Keycloak to enter a username and password in a new browser tab and get a response from the Quarkus application.
+Set a relative service endpoint path and click *Log in to your web application*.
+You are redirected to Keycloak to enter a username and password in a new browser tab before you get a response from the Quarkus application.
 
-Note that in this case Dev UI does not really enrich a dev experience since it is a Quarkus OIDC `web-app` application which controls the authorization code flow and acquires the tokens.
+In this case, the Dev UI does not enrich the developer experience.
```

Comment:
@sheilamjones May be delete that line but say `In this case the Quarkus OIDC web-app application controls the authorization code flow and acquires the tokens.` ?

Reply:
@sheilamjones Actually, this is what the next line already says it.

So, lets try `In this case, the Dev UI is not very helpful because the Quarkus OIDC web-app application controls the authorization code flow and acquires the tokens` and remove line `#196`

Reply:
Updated. Thanks!